apps/rbac/create-job-role.yaml

```diff
@@ -0,0 +1,28 @@
+# To allow the RD developers to create jobs in nonprod
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: create-job-role
+  namespace: ${NAMESPACE}
+rules:
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["create"]
+- apiGroups: ["batch"]
+  resources: ["cronjobs"]
+  verbs: ["create"]
```

Comment:
why? the request was just for jobs

Reply:
There was also many mentions of cronjobs in that ticket as well, so I added in case both were needed
Will remove though